### DISCUSSION TITLE
No sponsors?

### DISCUSSION THREAD

**Discussion by Trapper4888** [cline/cline#509] [Feature Requests]
Hey there! Have you noticed how your chats seem to rack up a ton of tokens on OpenRouter? And who knows how many more through other API providers! It's pretty impressive, really. 
You're definitely driving a lot of paid traffic to these platforms. 

So, why don't you have any official sponsors yet? It seems like you'd be a great candidate for some sponsorship deals!

**Comment by farbour** [comment]
+1, I'm pretty sure a lot of these guys would love to support the project and get some visibility

**Reply by ctwhome** [reply]
I'll be happy to pay as an individual. Cline has changed my way lf working forever.

**Comment by marketcalls** [comment]
Definitely Cline had changed the way we build trading applications. Love to pay for it if there is any sponsorship.
